src/ILLink.RoslynAnalyzer/DiagnosticTargets.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace ILLink.RoslynAnalyzer
+{
+	[Flags]
+	public enum DiagnosticTargets
```

Comment:
Any reason why we're splitting this out? I liked having the type grouped up with the only method that it's being used in.